### PR TITLE
[ENTESB-17668] - Workaround for  error on OCP3

### PR DESCRIPTION
**Pull request by mkralik3**
Ref: https://issues.redhat.com/browse/ENTESB-17668

### CODE REVIEW

File: update_ocp.sh [hunk 1]
```diff
@@ -128,6 +128,30 @@ check_syndesis() {
 
 # ==============================================================
 
+waiting_api_error_is_gone() {
+  set +e
+  output=$(oc get syndesis 2>&1 > /dev/null)
+  if [[ $output == *"Error from server (NotFound): Unable to list"* ]]; then
+    echo "Waiting (up to 10 minutes) due to ENTESB-17668"
```
Comment (phantomjinx) [inline update_ocp.sh]: Although accurate this should probably most explicit.

**Review by phantomjinx — APPROVED** [review]
Just the 1 change to make the error message more descriptive.

File: update_ocp.sh (hunk 1 above)
Comment (mkralik3) [reply update_ocp.sh]: Updated to : `Waiting (up to 10 minutes) until 'oc get syndesis' command recognize a new Syndesis api version (ENTESB-17668)`

**Comment by claudio4j** [comment]
I have seen this kind of error message when migrating from v1beta2 to v1beta3 as a result of using the `client.Status().Update()` however it is unknown for me why takes so long to have the new apigroup to report its readiness .
Thanks for providing the workaround Matej.

**Review by claudio4j — APPROVED** [review]

**Pull request merged**